1904-12-28.xml

```diff
@@ -0,0 +1,6056 @@
+                            <cell><measure unit="pt">70</measure></cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">72</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Hummos</cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">165</measure></cell>
+                            <cell>,, ,,</cell>
+                            <cell><measure unit="pt">175</measure></cell>
+                        </row>
+                    </table>
+                    <table rows="7" cols="3" xml:id="deg-ta-cema02">
+                        <head>Cereals in Boat at Sahel</head>
+                        <row>
+                            <cell>Wheat </cell>
+                            <cell>Ard.</cell>
+                            <cell><measure unit="ard">4500</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Beans</cell>
+                            <cell>,,</cell>
+                            <cell><measure unit="ard">700</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Lentils</cell>
+                            <cell>,,</cell>
+                            <cell><measure unit="ard">1000</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Barley</cell>
+                            <cell>,,</cell>
+                            <cell><measure unit="ard">1500</measure></cell>
+                        </row>
+                        <row>
+                            <cell>,, Dura Shami</cell>
+                            <cell>,,</cell>
+                            <cell><measure unit="ard">50</measure></cell>
+                        </row>
+                        <row>
+                            <cell>,, Rafia</cell>
+                            <cell>,,</cell>
+                            <cell><measure unit="ard">1000</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Helba</cell>
+                            <cell>,,</cell>
+                            <cell><measure unit="ard">150</measure></cell>
+                        </row>
+                    </table>
+                </div>
+                <div type="item" xml:id="deg-el-shli01">
+                    <head>SHARE LIST</head>
+                    <p>Issued by the "Association des Courtiers en Valeurs d'Alexandrie”.</p>
+                    <p>Clôture d'aujourd'hui à 12h.45 p.m.</p>
+                    <table rows="39" cols="5" xml:id="deg-ta-shli01">
+                        <row>
+                            <cell>Agric. Bank of Egypt</cell>
+                            <cell>Lst.</cell>
+                            <cell><measure
```

Comment:
move this `</div>` to the end of the whole cheap advertisements section. make all the divs it contains into "item" type, rather than cable.